internal/backend/local/test.go

```diff
@@ -1124,17 +1136,50 @@ func (runner *TestFileRunner) GetVariables(config *configs.Config, run *modulete
 // This function is essentially the opposite of AddVariablesToConfig which
 // makes the config match the variables rather than the variables match the
 // config.
-func (runner *TestFileRunner) FilterVariablesToConfig(config *configs.Config, values terraform.InputValues) terraform.InputValues {
+//
+// warnOnSensitivityChange will prompt this function to add warnings if an input
```

Comment:
Is this a name that was refactored out elsewhere?

Reply:
yeah, initially I had an argument that meant the function would only selectively add the warnings if this was true but then I realised the caller could just ignore the diagnostics so we didn't need to modify the function to exclude them internally.

I must've forgot update the doc comment when I removed it!